embed-code-go/embedding/parsing/parsing_context.go

```diff
@@ -198,20 +207,13 @@ func (pc ParsingContext) readEmbeddingResult(
 
 // Returns the content of a file placed at filepath as a list of strings.
 func readLines(filepath string) []string {
-	file, err := os.Open(filepath)
+	bytes, err := os.ReadFile(filepath) // just pass the file name
```

Comment:
I don't understand this comment. Since you pass `filePath` here instead of file name.